guides/slack-guide.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: 'How to use Novu to send notifications to a Slack channel'
+description: 'Leverage the integration store to send Slack notifications'
+icon: 'circle-4'
+---
+
+# Introduction
+
+In this guide, you'll learn how to use Novu to send notifications directly to a Slack channel. But before coding anything up, we first need to go through a setup process. 
+You can find the corresponding docs for the setup [here](https://docs.novu.co/channels-and-providers/chat/slack).
+<Note>The entire code of this app (frontend as well as backend) can be found [here](https://github.com/novuhq/slack-app).</Note>
```

Comment:
We agreed not to use here anymore. Please let's stick to the standards we have set.